### PR TITLE
API-7909: configuration-api bots <-> agents split

### PR DESCRIPTION
**Pull request by lpmichalik**
## 🚀 Links

- [Feature branch](https://developers.labs.livechat.com/docs/feature/API-7909-split-bots)
- [Jira](https://livechatinc.atlassian.net/browse/API-7909)

## 📓 Description
https://livechatinc.atlassian.net/wiki/spaces/PAT/pages/1460044276/Split+agents+and+bots

## ✅ Checklist (for API docs)

- Changelog
- API versions
- Web & RTM
- **Table:** Available methods/webhooks/pushes
- **Table:** Specifics

For more guidelines, see [Updating API docs](https://livechatinc.atlassian.net/wiki/spaces/PAT/pages/761529400/Updating+API+docs).

### Content

- New content
- Proofreading/content fixes
  
### Features (for the website)

- New feature
- Bug fix
- Breaking change

### CODE REVIEW

File: content/management/configuration-api/v3.3/index.mdx [hunk 1]
```diff
@@ -1238,15 +1238,28 @@ Returns the list of Bots created within a license. The method behavior differs d
...
+
+| Possible value        | Notes                                                     |
+| --------------------- | --------------------------------------------------------- |
+| `groups`              | Groups an Agent belongs to                                |
+| `work_scheduler`      | Work scheduler object for an Agent                        |
+| `email_subscriptions` | A list of the Agent's active email subscriptions          |
+| `notifications`       | A list of the Agent's enabled notifications               |
+| `job_title`           | Agent's job title                                         |
+| `mobile`              | Agent's mobile number                                     |
+| `max_chats_count`     | Agent's maximum number of concurrent chats                |
```
Comment (janowsiany) [inline content/management/configuration-api/v3.3/index.mdx:1261]: refer notes to bot not an agent, same for get_bot

File: content/management/changelog/index.mdx [hunk 2]
```diff
@@ -18,6 +18,8 @@ Since the migration is still ongoing, different parts will be successively moved
 ### Agents
 
 - In [**Create Agent**](/management/configuration-api/v3.3/#create-agent), there's a new notification option, `incoming_message_for_focused_chat`.
+- [**Get Agent**](/management/configuration-api/v3.3/#get-agent) method may no longer be used to fetch bots.
+- [**List Agents**](/management/configuration-api/v3.3/#list-agents) method will no longer return bots.
```
Comment (janowsiany) [inline content/management/changelog/index.mdx:22]: can we please unify these and make wording the same?

File: content/management/configuration-api/v3.3/index.mdx [hunk 3]
```diff
@@ -1286,9 +1311,22 @@ curl -X POST \
...
+
+| Possible value        | Notes                                                     |
+| --------------------- | --------------------------------------------------------- |
+| `work_scheduler`      | Work scheduler object for an Agent                        |
+| `groups`              | Groups an Agent belongs to                                |
+| `email_subscriptions` | A list of an Agent's active subscriptions                 |
+| `notifications`       | A list of an Agent's enabled notifications                |
+| `job_title`           | Agent's job title                                         |
+| `mobile`              | Agent's mobile number                                     |
+| `max_chats_count`     | Agent's maximum number of concurrent chats                |
```
Comment (janowsiany) [inline content/management/configuration-api/v3.3/index.mdx:1329]: webhooks?

Comment (janowsiany) [inline content/management/configuration-api/v3.3/index.mdx:1329]: what about webhooks?

File: content/management/configuration-api/v3.3/index.mdx [hunk 4]
```diff
@@ -1267,6 +1280,18 @@ curl -X POST \
 
 <CodeResponse type="configuration" version="v3.3" json="listBots"/>
 
+```json
```
Comment (ale-kacperczyk) [inline content/management/configuration-api/v3.3/index.mdx:1283]: This whole example should be removed:
image
You need to update the payload in `payloads/management/v3.3/configuration-api/responses/bots/listBots.json`

File: content/management/configuration-api/v3.3/index.mdx [hunk 5]
```diff
@@ -1310,6 +1348,26 @@ curl -X POST \
 
 <CodeResponse type="configuration" version="v3.3" json="getBot"/>
 
+```json
```
Comment (ale-kacperczyk) [inline content/management/configuration-api/v3.3/index.mdx:1351]: This whole example should be removed: 
image

Instead, update the payload in payloads/management/v3.3/configuration-api/responses/bots/getBot.json

**Review by ale-kacperczyk — APPROVED** [review]

**Review by janowsiany — APPROVED** [review]
👍

**Comment by janowsiany** [comment]
just wait for implementation of these changes before putting this on prod

**Review by janowsiany — APPROVED** [review]

**Review by ale-kacperczyk — APPROVED** [review]

**Pull request merged**